packages/next/build/webpack-config.ts

```diff
@@ -1007,7 +1007,12 @@ export default async function getBaseWebpackConfig(
         ...(!config.images.disableStaticImages && isWebpack5
           ? [
               {
-                test: /\.(png|svg|jpg|jpeg|gif|webp|ico|bmp)$/i,
+                // Exclude svg if the user defined a custom webpack config
+                // because it might include the `@svgr/webpack` plugin or
+                // the `babel-plugin-inline-react-svg` plugin.
+                test: config.webpack
```

Comment:
Should we look for the specific configuration that matches these imports similar to how we check for existing CSS configuration and disable the built-in config if custom config is detected?

Reply:
@ijjk Yeah that sounds good. How does this look?

```
const match = config.module.rules.find(rule => rule.test && rule.test.test('.svg'))
```

Reply:
Yeah like that, here's [the CSS check](https://github.com/vercel/next.js/blob/51022d5819c382831a65f963dcf4e69f46072bc9/packages/next/build/webpack-config.ts#L1562-L1565) for reference

Reply:
Done, thanks! 👍